serialization/astronomy.proto

```diff
@@ -17,7 +17,7 @@ message GravityModel {
     optional string reference_angle = 8;
     optional string angular_frequency = 9;
     // Oblate body.
-    optional string j2 = 5;
+    optional string j2 = 5;  // TODO(phl): Change to a double.
```

Comment:
It's probably more convenient to keep it a string, since these protos mirror KSP config nodes, and I wouldn't trust KSP's ability to keep floating-point numbers invariant under serialization/deserialization.